### PR TITLE
Condafy meme and fimo

### PR DESCRIPTION
**Pull request by gregvonkuster**
and update requirements to meme 4.11.2 and graphicsmagick to 1.3.23

### CODE REVIEW

**Comment by gregvonkuster** [comment]
Can this be merged?  It's holding up some things I'm working on. ;)

File: tools/meme/fimo.xml [hunk 1]
```diff
@@ -1,66 +1,57 @@
-<tool id="meme_fimo" name="FIMO" version="4.11.1.0">
+<tool id="meme_fimo" name="FIMO" version="4.11.2.0">
     <description>- Scan a set of sequences for motifs.</description>
     <requirements>
-        <requirement type="package" version="1.3.20">graphicsmagick</requirement>
-        <requirement type="package" version="4.11.1">meme</requirement>
+        <requirement type="package" version="1.3.23">graphicsmagick</requirement>
+        <requirement type="package" version="4.11.2">meme</requirement>
```
Comment (nsoranzo) [inline tools/meme/fimo.xml]: Macro for requirements?

File: tools/meme/fimo.xml [hunk 2]
```diff
@@ -1,66 +1,57 @@
...
-            --html_output "${html_outfile}"
-            --interval_output '${interval_outfile}'
-            --txt_output "${txt_outfile}"
-            --xml_output "${xml_outfile}"
-            --gff_output "${gff_outfile}"
-        ]]>
-    </command>
+    <command detect_errors="exit_code"><![CDATA[
+mkdir -p output &&
+python $__tool_directory__/fimo_wrapper.py
```
Comment (nsoranzo) [inline tools/meme/fimo.xml]: Single-quotes missing around `$__tool_directory__/fimo_wrapper.py`

File: tools/meme/fimo.xml [hunk 3]
```diff
@@ -1,66 +1,57 @@
...
+--input_motifs "${input_motifs}"
```
Comment (nsoranzo) [inline tools/meme/fimo.xml]: Can you use single quotes here and for the other data params?

**Comment by gregvonkuster** [comment]
Thanks @nsoranzo - should be fixed.  Sorry, I keep forgetting to pay closer attention to the command line quotes for these "newer" tools (i.e. about 1 year or less in this repo).  I'll try hard to to not keep making this mistake.  ;)

File: tools/meme/meme.xml [hunk 4]
```diff
@@ -1,87 +1,78 @@
...
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+meme '$input1'
+-o '${html_outfile.files_path}'
+-nostatus
+-maxsize 1000000
+#if str( $options_type.options_type_selector ) == 'advanced':
+    -sf '${ str( $options_type.sf ).replace( ' ', '_' ) }'
```
Comment (nsoranzo) [inline tools/meme/meme.xml]: I don't have time to test now (at GCC2017), but I'm worried the single quotes inside `replace( )` may not work any more.

**Comment by nsoranzo** [comment]
@gregvonkuster No need to apologise at all, that's why we have reviews!

**Comment by gregvonkuster** [comment]
@nsoranzo fixed - thanks again! ;)

**Pull request merged**